addons/web/static/src/views/kanban/kanban_controller.scss

```diff
@@ -227,6 +227,71 @@
     
         // Records colours
         @include o-kanban-record-color;
+
```

Comment:
Those rules don't apply, do they? The menu is never inside the renderer as it is a dropdown (rendering in the overlay container). That's why those rules aren't scoped in the o_legacy_kanban_record in the other file.

So I suggest the following:
 - here: move this whole block top level
 - in `web.KanbanRecordMenu` template, generate a different classname (e.g. `.o-dropdown--legacy-kanban-record-menu`) if the record is legacy (archInfo.legacyArch)
 - in kanban_dashboard.scss:  change the selector to use that legacy class

Feel free to reach me on discord if you have any question

Reply:
Hello @aab-odoo , I have made the changes. Could you please verify if they are correct? Kindly let me know if there are any mistakes or if anything needs to be corrected.

Thank you